test/unit/text/shaping.cpp

```diff
@@ -65,39 +65,39 @@ TEST_CASE("shaping")
     mapnik::face_manager fm(fl, font_file_mapping, font_memory_cache);
 
     {
-        std::vector<std::tuple<unsigned, unsigned>> expected =
-            {{{0, 0}, {0, 3}, {0, 4}, {0, 7}, {3, 8}, {11, 9}, {68, 10}, {69, 11}, {70, 12}, {12, 13}}};
+        std::vector<std::pair<unsigned, unsigned>> expected =
+            {{0, 0}, {0, 3}, {0, 4}, {0, 7}, {3, 8}, {11, 9}, {68, 10}, {69, 11}, {70, 12}, {12, 13}};
         // with default NotoSans-Regular.ttc and NotoNaskhArabic-Regular.ttf ^^^
```

Comment:
My bad, I thought I had used it in another test previously, turns out I had it stashed so not really tested on multiple compilers. Now I checked and it works for me with both gcc and clang (but I think it's using libstdc++ as well).